ports/geckolib/glue.rs

```diff
@@ -535,12 +545,12 @@ pub extern "C" fn Servo_AnimationCompose(raw_value_map: RawServoAnimationValueMa
         return;
     }
 
-    let position = unsafe {
+    let progress = unsafe {
```

Comment:
That's because it's a different  meaning.  'progress' here is just a progress for whole duration, Whereas 'position' is a ratio value in a keyframe corresponding to the progress.